### ISSUE TITLE
Add ability to login with a token

### ISSUE THREAD

**Issue by tommoor** [speak/account#37]
Most people coming from go.speak.io will have an account without an email address so won't be able to login - we need to create a method for them to get a single-use token to login... thoughts?

**Comment by awsmsrc** [comment]
alternatively we could us cookies?

They are on the same domain so we could just auto log them in?

**Comment by tommoor** [comment]
Yep, happy with that - I'm thinking we pretty much force folks to add an email/password if they go to the account area - probably persistent warning banners or something
